designer/server/src/main/resources/defaultDesignerConfig.conf

```
@@ -210,6 +210,19 @@ testDataSettings: {
   resultsMaxBytes: 50000000
 }
 
+scenarioLabelSettings: {
+  validationRules = [
+    {
+      validationPattern: "^[a-zA-Z0-9-_]+$",
+      validationMessage: "Scenario label can contain only alphanumeric characters, '-' and '_'"
+    }
+    {
+      validationPattern: "^.{1,4}$"
```

```suggestion
      validationPattern: "^.{1,20}$"
```
is it merged with dev-application.conf? can we throw this config from dev-application.conf?